pkg/ovs/openflow/ofctrl_builder.go

```diff
@@ -231,13 +231,16 @@ func (b *ofFlowBuilder) MatchCTStateSNAT(set bool) FlowBuilder {
 }
 
 func (b *ofFlowBuilder) MatchCTMark(mark *CtMark) FlowBuilder {
-	ctmarkKey := fmt.Sprintf("ct_mark=0x%x", mark.value)
-	b.ofFlow.Match.CtMark = mark.value
```

Comment:
I think you could simply change line 235 as `b.ofFlow.Match.CtMark = mark.GetValue()`?

Reply:
Thanks, updated.